taichi/rhi/metal/metal_device.h

```diff
@@ -1,18 +1,21 @@
 #pragma once
-#include <memory>
 #include "taichi/common/logging.h"
 #include "taichi/rhi/device.h"
-#include "taichi/rhi/metal/metal_api.h"
 #include "taichi/rhi/impl_support.h"
+#include "taichi/rhi/metal/metal_api.h"
+#include <memory>
 
 #if defined(__APPLE__) && defined(__OBJC__)
+#import <CoreGraphics/CoreGraphics.h>
 #import <Foundation/Foundation.h>
 #import <Metal/Metal.h>
 #import <MetalKit/MetalKit.h>
-#import <CoreGraphics/CoreGraphics.h>
+#import <QuartzCore/QuartzCore.h>
```

Comment:
Why is this replaced?

Reply:
nvm the order changed it appears

Reply:
it's better to not change order of things unless it's a formatting / style related PR

Reply:
I don't know, but the CoreGraphics header is still included in that file, just earlier